org.eclipse.winery.frontends/app/tosca-management/src/app/header/wineryOauth/wineryOAuth.service.ts

```diff
@@ -87,33 +93,43 @@ export class WineryOAuthService {
      * Logs the user out by deleting all saved key values paris in the local storage.
      */
     logout() {
-        this.storage.removeItem(StorageElements.state);
-        this.storage.removeItem(StorageElements.accessToken);
-        this.storage.removeItem(StorageElements.tokenType);
-        this.storage.removeItem(StorageElements.userName);
+        this.http
+            .post(
+                backendBaseURL + '/admin/githublogout', {},
+            )
+            .subscribe(
+                () => this.handleLogout(),
+                (error: any) => this.handleError(error)
+            );
     }
 
     getUserInformation() {
-        if (isNullOrUndefined(this.storage.getItem(StorageElements.accessToken))) {
+        if (isNullOrUndefined(this.config.configuration.git.accessToken)) {
```

Comment:
`isNullOrUndefined` ist veraltet. Einfach nur `if (!this.config.configuration.git.accessToken)`

Das bitte überall entsprechend rauswerfen.